internal/iam/repository_scope.go

```diff
@@ -85,9 +86,9 @@ func (r *Repository) CreateScope(ctx context.Context, s *Scope, userId string, o
 	var adminRoleRaw any
 	switch {
 	case userId == "",
-		userId == "u_anon",
-		userId == "u_auth",
-		userId == "u_recovery",
+		userId == perms.AnonymousUserId,
+		userId == perms.AdminUserId,
```

Comment:
I think you meant for this to be `perms.AnyAuthenticatedUserId` or `perms.AnyAuthedUserId`